test/resources/common.resource

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Library    OperatingSystem
+Library    String
+Resource   ../resources/kubeconfig.resource
+
+*** Keywords ***
+Setup Suite
```

Comment:
Was using this PR as some sort of guinea pig to test the CI job. Now that I got that running (meaning junit and such) it contains the bare minimum to run at least one test.

I expect the resource files to undergo massive changes as more tests are added. These are only placeholders to get things going while we acquire more knowledge about the framework.
I am all for composition, so the more primitive-like these files look, the better. We will see if they evolve into more complex functions once we add more tests (it might be that almost all setup/teardown strategies end up being the same).

I would be happy to merge this and then keep modifying it with new test suites.

Reply:
Absolutely, yeah.